unit-tests/live/d400/test-d405-calibration-stream.py

```diff
@@ -0,0 +1,88 @@
+# License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
Add negative tests in multiple configuration (Y16 + RGB at VGA)

Reply:
issues to be checked with FW team